### PR TITLE
Announcement for Ivanchuk Beta1 release

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette can you review/merge.

**Comment by JPrause** [comment]
all smoke tests passed

File: site/_posts/2019-07-26-Announcing-Icanchuk-Beta1.html.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Add option to schedule automation requests via workflow
+* Add a tool to assist in reconnecting vms
+* Add zone to service provisioning
+* Add method to copy service templates
+* Allow user to select Network Port for provisioning
+* Add support for exporting and importing reports
+
+Take a look at the [changelog](https://github.com/ManageIQ/manageiq/blob/ivanchuk/CHANGELOG.md/) for more details.
+
+We encourage you to [download](http://manageiq.org/download/) the Hammer Beta Release and try it out!
```
Comment (cybette) [inline site/_posts/2019-07-26-Announcing-Icanchuk-Beta1.html.md]: s/Hammer/Ivanchuk/ :wink:

Comment (JPrause) [reply site/_posts/2019-07-26-Announcing-Icanchuk-Beta1.html.md]: Thanks @cybette I had missed one prior, caught it, and thought, "there couldn't possibly be any others".

Comment (cybette) [reply site/_posts/2019-07-26-Announcing-Icanchuk-Beta1.html.md]: been there, done that :P

**Comment by cybette** [comment]
LGTM. Will merge Monday morning as discussed. Thanks!

**Comment by simaishi** [comment]
@JPrause As I mentioned... there is no Docker image for Ivanchuk Beta1, so it shouldn't be listed in the download page.

**Pull request merged**